ui-v2/src/api/work-pools/work-pools.ts

```diff
@@ -106,16 +106,11 @@ export const buildFilterWorkPoolsQuery = (
 export const buildCountWorkPoolsQuery = (filter: WorkPoolsCountFilter = {}) =>
 	queryOptions({
 		queryKey: queryKeyFactory.count(filter),
-		queryFn: async () => {
+		queryFn: async (): Promise<number> => {
```

Comment:
It might be nice if we required return value typing on these query functions instead of relying on inference